main/client/src/mill/main/client/lock/FileLocked.java

```diff
@@ -2,7 +2,7 @@
 
 class FileLocked implements Locked {
 
-    private java.nio.channels.FileLock lock;
+    protected java.nio.channels.FileLock lock;
```

Comment:
Do we really need to expose this mutable variable? Can we make it `final` or provide a `protected` getter?

Reply:
?

Reply:
opening a separate PR to finalize this and a bunch of other things that probably should be final

Reply:
final

Reply:
https://github.com/com-lihaoyi/mill/pull/3366